source/adapters/level_zero/device.hpp

```diff
@@ -160,7 +160,7 @@ struct ur_device_handle_t_ : _ur_object {
   // provide support for only one, like for Intel(R)
   // Data Center GPU Max, for which L0 driver only
   // supports stateless.
-  int32_t useOptimized32bitAccess();
+  int32_t useRelaxedAllocationLimits();
```

Comment:
`bool`?

Reply:
Previously we had the possible values of 0, 1, and -1, but with the change you are right the new values are 0,1 true/false so I can change that logic to be cleaner.

Reply:
Updated to use bool instead of the int32_t and updated the checks to simplify to true/false.